conf/app.routes

```diff
@@ -1,7 +1,7 @@
 # microservice specific routes
 
 GET     /                               uk.gov.hmrc.agentsubscriptionfrontend.controllers.StartController.root
```

Comment:
I'm not sure, but should the / 'root' also take the continue parameter and preserve it when redirecting to /start 'start'?